.cirrus.yml

```diff
@@ -736,6 +736,33 @@ upgrade_test_task:
     always: *logs_artifacts
 
 
+image_build_task:
+    name: "Build multi-arch images and manifest lists"
+    alias: image_build
+    skip: *tags
+    # FIXME: Run nightly on 'main' branch
+    # only_if: $CIRRUS_CRON == 'multiarch'
+    only_if: $CIRRUS_CHANGE_TITLE =~ '.*CI:BUILD.*'
+    depends_on:
+        - ext_svc_check
+    timeout_in: 120m  # emulation is sssllllooooowwww
+    gce_instance:
+        image_project: libpod-218412
+        zone: "us-central1-a"
+        # More muscle required for parallel multi-arch build
+        type: "n2-standard-4"
```

Comment:
Is this the magic setting that precludes using `*standardvm`? If it's not possible to reuse `standardvm`, it'd be nice to document why

Reply:
Nope, you're right I could use that alias and only update the `type` line.  The "why" is in the comment above: More muscle is required, multi-arch emulation is incredibly CPU intensive.